docs/application/native/guides/app-management/tidl.md

```diff
@@ -19,9 +19,13 @@ Usage:
   -o, --output=OUTPUT         The generated interface file.
   -p, --proxy                 Generate proxy code.
   -s, --stub                  Generate stub code.
+  -r, --rpclib                Generate C# library for rpc-port.
+  -t, --thread                Generate thread code with stub code.
```

Comment:
Generates thread code with stub code.

Reply:
We don't usually use the third person singular in usage comments of the tool.

Reply:
@hjhun, I shall wait to hear Catherine's opinion on this. I agree that consistency is also a valid point as you pointed out.

@mijin85cho, could you suggest what should be the usage?
From language point of view either we say:

To generate the proxy code.
or
Generates proxy code.

And if we change here then we have to change for all options.

Or do we leave as it is for consistency sake?

Reply:
@annie-samsung As this is kind of a help message, I think "To generate" or "Generates" is less proper for the help message format on command window. It is less related with the consistency, but with the help message format.

Below is an example from git help message on terminal window on my local PC. Please refer it.

![image](https://user-images.githubusercontent.com/9958374/91940497-7c6eb800-ed32-11ea-9234-c1cf61970c17.png)

Reply:
@mijin85cho, @hjhun thanks for your responses.
I am guessing it is how they decided to follow for GIT. If you see Windows help commands, they all follow third person singular usage. Similarly ADB (Android Debugger) help commands they also follow third person singular usage. 

Since we have been following this style it makes sense to leave it as is. I will keep this in mind.
I have no further comments. Hence approving this PR.

Reply:
@annie-samsung Thanks for the comment. It seems some tools are using third-person singular and others are not as you said. In this description, we are not using third-person singular form and the added options are also following the existing style.